jujugui/static/gui/src/app/config-debug.js

```diff
@@ -1,67 +0,0 @@
-/*
```

Comment:
Both config-* files were part of the old rollup system. We don't need them anymore since config.js is served from the wsgi app.